index.d.ts

```diff
@@ -76,7 +76,9 @@ export type HTMLRendererConfig = BaseRendererConfig & {
     hideOnTransparent?: boolean;
 };
 
-export type AnimationConfig<T extends 'svg' | 'canvas' | 'html' = 'svg'> = {
+export type RendererType = 'svg' | 'canvas' | 'html';
+
+export type AnimationConfig<T extends RendererType> = {
```

Comment:
@wemyss removing default template type here is a breaking change and projects which import `AnimationConfig` without further `<'svg'>` are now failing.

Fix proposed in #2774